src/components/layermanager/partials/layermanager.html

```diff
@@ -2,7 +2,7 @@
   <li ng-repeat="layer in filteredLayers" class="ga-layer-folded"
       ng-class="{
         'ga-layer-time-enabled': layer.timeEnabled,
-        'ga-layer-warning': layer.useThirdPartyData
+        'ga-layer-warning': layer.useThirdPartyData || layer.externalStyleUrl
```

Comment:
Set the `useThirdPartyData` in the gaLayers Service just after you set `externalStyleUrl`